lib/gooddata/lcm/actions/synchronize_etls_in_segment.rb

```diff
@@ -81,13 +81,42 @@ def call(params)
             end
           end
 
-          params.synchronize.peach do |info|
+          delete_extra_process_schedule = GoodData::Helpers.to_boolean(params.delete_extra_process_schedule)
+
+          params.synchronize.peach do |info| # rubocop:disable Metrics/BlockLength
```

Comment:
I believe that code performance is unrelated to coding style. The BlockLength rule makes a perfect sense here as the method is getting very lengthy. One way to go about this would be extracting the bodies of the `if`s into private methods.